js/script.js

```diff
@@ -1,2 +1,25 @@
-let p = document.querySelector ("p")
-p.innerHTML = "Hello"
+document.querySelector('#push').onclick = function(){
+    if(document.querySelector('#input').value.length == 0){
+        alert("Kindly Enter Task Name!!!!")
+    }
+
+    else{
+        document.querySelector('#tasks').innerHTML += `
+            <div class="task">
+                <span id="taskname">
+                    ${document.querySelector('#input').value}
+                </span>
+                <button class="delete">
+                    <i class="far fa-trash-alt"></i>
+                </button>
+            </div>
+        `;
```

Comment:
Consider creating a whole node first and then add with the appendChild method to the parent class element.
For more information on this, please follow https://developer.mozilla.org/en-US/docs/Web/API/Node/appendChild